### PR TITLE
Added Toggle Tail config in Debug.

### PR DESCRIPTION
**Pull request by sjoshid**
PR for https://github.com/xi-editor/xi-editor/issues/922

### CODE REVIEW

File: Sources/XiEditor/XiCore.swift [hunk 1]
```diff
@@ -109,6 +111,10 @@ final class CoreConnection: XiCore {
         let params = ["destination": destination, "frontend_samples": frontendSamples] as AnyObject
         sendRpcAsync("save_trace", params: params)
     }
+    
+    func toggleTailConfig(enabled: Bool) {
```
Comment (mmatoszko) [inline Sources/XiEditor/XiCore.swift]: Could you add an unit test for that?
[`testTracingConfigEnabled`](https://github.com/xi-editor/xi-mac/blob/master/Tests/XiEditorTests/XiCoreTests.swift#L37) should be a clear example how to do it 😄

File: Sources/XiEditor/AppDelegate.swift [hunk 2]
```diff
@@ -490,6 +499,10 @@ class AppDelegate: NSObject, NSApplicationDelegate, XiClient {
     func updateRpcTracingConfig(_ enabled: Bool) {
         xiCore?.tracingConfig(enabled: enabled)
     }
+    
+    func updateToggleTailConfig(_ enabled: Bool) {
```
[the remark this answers is not in the document]
Comment (sjoshid) [reply Sources/XiEditor/AppDelegate.swift]: Done.

File: Sources/XiEditor/XiCore.swift (hunk 1 above)
Comment (sjoshid) [reply Sources/XiEditor/XiCore.swift]: Done.

**Comment by jansol** [comment]
The function and RPC are called 'toggle' but what it does looks more like 'set' at a quick glance. This seems somewhat confusing to me. Maybe rename or explain the naming in a comment in the code?

**Comment by cmyr** [comment]
This is tracking https://github.com/xi-editor/xi-editor/pull/1011; the final shape of this will be determined by decisions there, so holding review until then.

**Comment by sjoshid** [comment]
@cmyr 
Made some progress here. I have followed existing language changed implementation as it is similar in functionality to tail feature.

**Comment by cmyr** [comment]
@sjoshid This includes a bunch of files I don't think you intended?

A tip I find useful: after updating a PR (and before confirming to open a PR) click on the "files changed" tab and read through the diffs. I very often find mistakes I've made this way!

**Comment by sjoshid** [comment]
Oops. Sorry about that. Not sure how they sneaked in. I think I did a bunch of useless commits but the final version looks good.

**Pull request closed without merging**